.dockerignore

```diff
@@ -122,6 +122,9 @@ docs/_build/
 docs/_api/
 docs/_doctrees/
 
+# Exclude tmp directory
+tmp
```

Comment:
Should this include all directories named `tmp` or just in the repo root?

Reply:
just the root one - yeah. I added it hastily :)